app/login.tsx

```diff
@@ -161,6 +161,16 @@ export default function LoginScreen() {
             <Text style={styles.signUpLink}>Sign Up</Text>
           </Pressable>
         </View>
+
+        {/* Test Onboarding Button */}
+        <View style={styles.testContainer}>
+          <TouchableOpacity
+            style={styles.testButton}
+            onPress={() => router.push('/onboarding-1')}
+          >
+            <Text style={styles.testButtonText}>Test Onboarding</Text>
+          </TouchableOpacity>
```

Comment:
The sign up button is the one that will navigate to the `onboarding` use that button instead of this